packages/react/__tests__/rehydration.js

```diff
@@ -166,3 +197,131 @@ test('initializing another Emotion instance should not move already moved styles
     </head>
   `)
 })
+
+test('xxx', () => {
```

Comment:
we need to write a proper title here 😅

Reply:
```suggestion
test('dynamic global styles in SSR should work as expected', () => {
```

Maybe something like this?

Reply:
⚠️ this was marked as resolved, but doesn't seem to be

Reply:
ah, ye - I forgot to push out the change I've applied locally 😅 good, vigilant eye!